modelservice/src/main/java/com/expedia/adaptivealerting/modelservice/web/AnomalyController.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2018-2019 Expedia Group, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.expedia.adaptivealerting.modelservice.web;
+
+import com.expedia.adaptivealerting.core.anomaly.AnomalyResult;
+import com.expedia.adaptivealerting.modelservice.service.AnomalyRequest;
+import com.expedia.adaptivealerting.modelservice.service.AnomalyService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+@RestController
+public class AnomalyController {
+
+    @Autowired
+    private AnomalyService anomalyService;
+
+    @PostMapping(path = "/api/anomalies", consumes = "application/json", produces = "application/json")
```

Comment:
If we use the `/api` prefix, let's attach that to the controller itself so we don't have to repeat it for each handler method.

Reply:
done